src/class-buddypress.php

```diff
@@ -330,7 +330,7 @@ private function setup_globals() {
 
 		/** Versions */
 		$this->version    = defined( 'BP_PLATFORM_VERSION' ) ? BP_PLATFORM_VERSION : ( defined( 'BP_VERSION' ) ? BP_VERSION : '1.0.0' );
-		$this->db_version = 16401;
+		$this->db_version = 16459;
```

Comment:
@asaquzzamanbb update the DB version as video have.

Reply:
@KartikSuthar 
video which one.
do you asking about this version `$raw_db_version < 16601` check the following link.
https://github.com/buddyboss/buddyboss-platform/blob/video-uploading/src/bp-core/bp-core-update.php#L327